menoh/tensorrt/Inference.hpp

```diff
@@ -53,6 +60,15 @@ namespace menoh_impl {
             unique_ptr_with_destroyer<ICudaEngine> engine;
             unique_ptr_with_destroyer<IExecutionContext> context;
 
+#ifdef MENOH_ENABLE_TENSORRT_MODEL_CACHING
+            std::string calc_model_hash(
+              std::unordered_map<std::string, array> const& input_table,
+              std::unordered_map<std::string, array> const& output_table,
+              menoh_impl::model_data const& model_data, config const& config);
+
+            unique_ptr_with_destroyer<IRuntime> runtime;
```

Comment:
It seems that the `runtime` is used only in `Inference::Inference`.
Is it better to change it to a local variable in `Inference::Inference`?

Reply:
Right. I will modify so.